tests/swo/test_base_client.py

```
@@ -0,0 +1,103 @@
+from http import HTTPStatus
+
+import pytest
+import responses
+
+from swo_aws_extension.swo.base_client import OAuthSessionClient
+
+BASE_CLIENT_MODULE = "swo_aws_extension.swo.base_client"
+TEST_BASE_URL = "https://api.test.com/"
+TEST_OAUTH_URL = "https://oauth.test.com/token"
+TEST_CLIENT_SECRET = "client-secret"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "TEST_CLIENT_SECRET"

(S105)